<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGISWindow</class>
 <widget class="QWidget" name="TestGISWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>967</width>
        <height>478</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextBrowser" name="textBrowser_2">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. Нарушение &lt;span style=&quot; font-weight:600;&quot;&gt;целостности&lt;/span&gt; привет к&lt;span style=&quot; font-family:'Tahoma,arial,verdana'; font-size:8pt; color:#000000; background-color:#ffffff;&quot;&gt; последствиям в социальной, политической, международной, экономической, финансовой или иных областях деятельности и (или) информационная система и (или) оператор (обладатель информации)...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBox_4">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="text">
              <string>не могут выполнять возложенные на них функции</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_5">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>не могут выполнять хотя бы одну из возложенных на них функций</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_6">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>могут выполнять возложенные на них функции с недостаточной
эффективностью или выполнение функций возможно только
с привлечением дополнительных сил и средств</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1. Нарушение &lt;span style=&quot; font-weight:600;&quot;&gt;конфиденциальности&lt;/span&gt; привет к&lt;span style=&quot; font-family:'Tahoma,arial,verdana'; font-size:8pt; color:#000000; background-color:#ffffff;&quot;&gt; последствиям в социальной, политической, международной, экономической, финансовой или иных областях деятельности и (или) информационная система и (или) оператор (обладатель информации)...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_1">
           <item>
            <widget class="QCheckBox" name="checkBox_1">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>не могут выполнять возложенные на них функции</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>не могут выполнять хотя бы одну из возложенных на них функций</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>могут выполнять возложенные на них функции с недостаточной
эффективностью или выполнение функций возможно только
с привлечением дополнительных сил и средств</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextBrowser" name="textBrowser_3">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3. Нарушение &lt;span style=&quot; font-weight:600;&quot;&gt;доступности&lt;/span&gt; привет к&lt;span style=&quot; font-family:'Tahoma,arial,verdana'; font-size:8pt; color:#000000; background-color:#ffffff;&quot;&gt; последствиям в социальной, политической, международной, экономической, финансовой или иных областях деятельности и (или) информационная система и (или) оператор (обладатель информации)...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBox_7">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>не могут выполнять возложенные на них функции</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_8">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>не могут выполнять хотя бы одну из возложенных на них функций</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_9">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>могут выполнять возложенные на них функции с недостаточной
эффективностью или выполнение функций возможно только
с привлечением дополнительных сил и средств</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="accept">
         <property name="text">
          <string>Продолжить</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTextBrowser" name="textBrowser_4">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4. Масштаб информационной системы.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBox_10">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Федеральный</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_11">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Региональный</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_12">
             <property name="minimumSize">
              <size>
               <width>425</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Объектовый</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
